notifications.yaml

```diff
@@ -0,0 +1,268 @@
+                Location (URL) of accepted message resource.
+
+                A GET request to this URL returns the message status and
+                details.
+definitions:
+  ProblemJson:
+    title: Problem Type
+    type: object
+    properties:
+      type:
+        type: string
+        format: uri
+        description: |
+          An absolute URI that identifies the problem type.  When dereferenced,
+          it SHOULD provide human-readable documentation for the problem type
+          (e.g., using HTML).
+        default: 'about:blank'
+        example: 'https://example.com/problem/constraint-violation'
+      title:
+        type: string
+        description: |
+          A short, summary of the problem type. Written in english and readable
+          for engineers (usually not suited for non technical stakeholders and
+          not localized); example: Service Unavailable
+      status:
+        type: integer
+        format: int32
+        description: >
+          The HTTP status code generated by the origin server for this
+          occurrence
+
+          of the problem.
+        minimum: 100
+        maximum: 600
+        exclusiveMaximum: true
+        example: 503
+      detail:
+        type: string
+        description: |
+          A human readable explanation specific to this occurrence of the
+          problem.
+        example: Connection to database timed out
+      instance:
+        type: string
+        format: uri
+        description: >-
+          An absolute URI that identifies the specific occurrence of the
+          problem.
+
+          It may or may not yield further information if dereferenced.
+  MessageContent:
+    title: MessageContent
+    type: object
+    properties:
+      body_short:
+        type: string
+        description: Message content (short version).
+        minLength: 3
+        maxLength: 100
+      body_long:
+        type: string
+        description: Message content (long version).
+        minLength: 3
+    required:
+      - body_short
+  MessageDefinition:
+    title: MessageDefinition
+    type: object
+    properties:
+      to:
+        description: The fiscal code of the destination user.
+        type: string
+      dry_run:
+        type: boolean
+        description: >-
+          This parameter, when set to true, allows developers to test a request
+          without actually sending a message.
+
+
+          The default value is false.
+      time_to_live:
+        type: integer
+        description: >-
+          This parameter specifies how long (in seconds) the message should be
+          kept in storage if the device is offline. The maximum time to live
+          supported is 1 year, and the default value is 1 hour.
+        minimum: 3600
+        maximum: 31536000
+      content:
+        $ref: '#/definitions/MessageContent'
+    required:
+      - to
+      - content
+  MessageStatus:
+    title: MessageStatus
+    type: object
+    properties:
```

Comment:
What can the client do with that information? `body_short` should contain the essence of the message, it doesn't make sense to have a short version if it's missing any important piece of information - the message should be able to exist solely in the short version - the longer version is there just as an "improved representation" of the message. The caller should assume that only the `body_short` version may be delivered and that version should contain all the required information. We should make this very clear and explicit in the docs (and check wether the early adopters understand this requirement from the documentation).

In fact `body_short` is required and `body_long` is optional.